pantheon/src/main/java/tech/pegasys/pantheon/cli/subcommands/blocks/BlocksSubCommand.java

```diff
@@ -135,4 +132,100 @@ public void run() {
       }
     }
   }
+
+  /**
+   * blocks dump sub-command
+   *
+   * <p>Dump a specific block from storage
+   */
+  @Command(
+      name = "dump",
```

Comment:
Instead of `dump` use `export` - it is reflexive to `import` - and change the name throughout the PR.

Reply:
ok i will do that. I changed the pull request. here is the new link if you want to transfer your comments https://github.com/PegaSysEng/pantheon/pull/1641

Reply:
I just made the requested changes here https://github.com/PegaSysEng/pantheon/pull/1641